### PR TITLE
Hide new list icon when list item is empty

### PR DESCRIPTION
**Pull request by KickButtowski80**

### CODE REVIEW

File: src/views/List.vue [hunk 1]
```diff
@@ -11,6 +11,7 @@
         :states="states || globalPreferences.defaultStateGroup.states"
         @blur="saveItem(index, $event)"
         @input="addNewItem(index, $event)"
+        @delete="delItem(index, $event)"
```
Comment (KickButtowski80) [inline src/views/List.vue:14]: part of #96

File: src/views/List.vue [hunk 2]
```diff
@@ -93,6 +94,9 @@ export default {
         });
       }
     },
+    delItem(index, item) {
```
Comment (KickButtowski80) [inline src/views/List.vue:97]: part of #96

File: src/components/ListItem.vue [hunk 3]
```diff
@@ -41,6 +43,14 @@
           v-if="!readOnly && subList">
             <v-icon>mdi-shield-link-variant-outline</v-icon>
         </v-btn>
+        <v-btn
```
Comment (KickButtowski80) [inline src/components/ListItem.vue:46]: part of #96

**Review by KickButtowski80 — COMMENTED** [review]
This PR contains parts of #96. Do not merge until #96 is merged

**Comment by skamansam** [comment]
I am closing this PR as irrelevant.

**Pull request closed without merging**